doc/cephfs/mdcache.rst

```diff
@@ -0,0 +1,82 @@
+=================================
+CephFS Distributed Metadata Cache
+=================================
+While the data for inodes in a Ceph file system is stored in RADOS and
+accessed by the clients directly, inode metadata and directory
+information is managed by the Ceph metadata server (MDS).
```

Comment:
I'd suggest mentioning that the metadata is also stored in RADOS. I find that people new to Ceph often assume it has a completely separate metadata store (like Lustre, for example).